twine/generate/triggers.c

```diff
@@ -105,6 +108,13 @@ spindle_trigger_apply(SPINDLEENTRY *entry)
 			// Set the target as DIRTY
 			sql_executef(entry->generate->db, "UPDATE \"state\" SET \"status\" = %Q WHERE \"id\" = %Q", "DIRTY", id);
 		}
+
+		// Set the earliest activation date some time in the future
+		now = time(NULL);
+		now += 10; // TODO As for the next fetch in Anansi, this should be made configurable.
```

Comment:
Ah! That's the big question. We can make it configurable to be able to easily change the value but it will be anyhow almost impossible to find one perfect value that will fit all the cases.